crates/runtime/src/runtime/default.rs

```diff
@@ -42,6 +41,8 @@ where
 
     /// Builds an `AccountStorage` instance.
     storage_builder: Box<StorageBuilderFn>,
+
+    template_prices: HashMap<TemplateAddr, FuncPrice>,
```

Comment:
I think we need to make it a `static` variable (outside of `DefaultRuntime`)

We probably don't want to re-use the same `DefaultRuntime` instance throughout the process lifetime.
Doing so implies we trust the Wasm Runtime (currently Wasmer but possible other one) not to have any
internal memory leak, or introduced bugs due to heavy usage.

Creating a new `DefaultRuntime` should be negligible. I think that we are better of with creating a Runtime per usage at this point in time.

Reply:
I think we should wrap `Runtime`s in a `Context` of some sort. I don't feel comfortable using a global variable...

Reply:
Or maybe pass in a shared `HashMap` to the `Runtime`

Reply:
That's fine as well to pass to `DefaultRuntime#new` a dependency like:
`cache: &mut HashMap<TemplateAddr, FuncPrice>`

Reply:
> I think we should wrap `Runtime`s in a `Context` of some sort. I don't feel comfortable using a global variable...

That's understandable. However, it seems that SVM will need some initialization code in order to create the shared dependency (to be injected later for each `DefaultRuntime#new`)